### PR TITLE
this is a test for commit status

### PR DESCRIPTION
**Pull request by syphonetic**

### CODE REVIEW

**Comment by factionsypho** [comment]
Added

**Review by damithc — COMMENTED** [review]
Need to make some progress in your project @factionsypho 
Perhaps you forgot to push your code? 🤔

**Comment by damithc** [comment]
@factionsypho Good to see you making steady progress. keep up the pace and aim for a strong finish.

File: src/main/java/EventCommand.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            throw new DukeException("The date of a event cannot be empty.");
+        }
+        int index = 0;
+        for(String command : commandLine.split(" ")){
+            ++index;
+            if(command.equals("at")){
+                if(String.join(" ", Arrays.copyOfRange(commandLine.split(" "), index, commandLine.split(" ").length)).isEmpty()){
+                    throw new DukeException("The date of a event cannot be empty.");
+                }
+                else {
```
Comment (damithc) [inline src/main/java/EventCommand.java]: FYI, this style of braces is not compliant with the given coding standard.

**Pull request closed without merging**